roles/download/defaults/main.yml

```diff
@@ -62,8 +61,6 @@ etcd_image_repo: "quay.io/coreos/etcd"
 etcd_image_tag: "{{ etcd_version }}"
 flannel_image_repo: "quay.io/coreos/flannel"
 flannel_image_tag: "{{ flannel_version }}"
-flannel_cni_image_repo: "quay.io/coreos/flannel-cni"
```

Comment:
We need to keep the flannel-cni image as it contains the CNI plugin binaries.

It could potentially be converted into an init container, but it would still need all of the existing volumes and vars.

`install-cni` not originally implemented as an initContainer as an attempt to stay in sync with the reference deployment found at https://github.com/coreos/flannel-cni